xml/mysql_banners.xml

```diff
@@ -1035,14 +1192,103 @@
+    <example service.version="10.3.22">5.5.5-10.3.22-MariaDB-1ubuntu1-log</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="MariaDB"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MariaDB"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:mariadb:mariadb:{service.version}"/>
+    <param pos="0" name="os.vendor" value="Ubuntu"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:canonical:ubuntu_linux:10.04"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB-\d\:.*\+maria\~focal$" flags="REG_ICASE">
+    <description>MariaDB MariaDB running on a Ubuntu Focal Fossil (20.4)</description>
+    <example service.version="10.5.2">5.5.5-10.5.2-MariaDB-1:10.5.2+maria~focal</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="MariaDB"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MariaDB"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:mariadb:mariadb:{service.version}"/>
+    <param pos="0" name="os.vendor" value="Ubuntu"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="os.version" value="20.04"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:canonical:ubuntu_linux:20.04"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB-mariadb\d\Smaverick$" flags="REG_ICASE">
+    <description>MariaDB MariaDB running on a Ubuntu Maverick (10.10)</description>
+    <example service.version="5.5.29">5.5.29-MariaDB-mariadb1~maverick</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="MariaDB"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MariaDB"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:mariadb:mariadb:{service.version}"/>
+    <param pos="0" name="os.vendor" value="Ubuntu"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="os.version" value="10.10"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:canonical:ubuntu_linux:10.10"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4}(?:\-\d)?)-MariaDB-alt(\d{1,2})(?:-log)?.*$" flags="REG_ICASE">
+    <description>MariaDB MariaDB running on a ALT Linux</description>
+    <example service.version="10.1.29" os.version="1">5.5.5-10.1.29-MariaDB-alt1.M80P.1</example>
+    <example service.version="10.2.15" os.version="2">5.5.5-10.2.15-MariaDB-alt2.M80P.3.S1</example>
```

Comment:
Can you verify the OS version?  The current version of Alt appears to be 9.0.  ALT 1.1 appears to have been released in 2001 and is unlikely to be running MariaDB.

https://en.altlinux.org/ALT
https://distrowatch.com/table.php?distribution=alt

Reply:
Good spot that would make sense that it would be unlikely that someone would be using that version, now im not sure if i can verify the OS from the banner, is it ok to remove the os values and just assert it as mariadb and leave the description as it being alt? or are we ok to fingeprint as alt without the version?

Reply:
I would remove the capture for `os.version`, the example test, and it's value below. I would remove `{os.version}` from the `os.cpe23` line but leave the rest.